content/en/docs/spinnaker-user-guides/kustomize-manifests.md

```diff
@@ -74,50 +40,30 @@ Define the artifact:
 ​
 You can now run your pipeline and get a Kustomize rendered manifest!
 ​
-## Kustomize in 2.17 (Beta)
-​
+## Kustomize 
 ### Requirements
-Kustomize in 2.17+ requires the [git/repo](https://www.spinnaker.io/reference/artifacts/types/git-repo/) artifact type.
-​
-### Enable Kustomize
-​
-Kustomize can be enabled by a feature flag in Armory 2.16 and later.
+Kustomize requires the git/repo artifact type.
 
-**Operator**
+**Configure git/repo artifact with Operator**
 
-Add the following settings to the `SpinnakerService` manifest and apply the changes:
+Add the follwing settings to the `SpinnakerService` manifest:
 
 ```yaml
-apiVersion: spinnaker.armory.io/{{< param operator-extended-crd-version >}}
+apiVersion: spinnaker.armory.io/v1alpha2
```

Comment:
```suggestion
apiVersion: spinnaker.armory.io/{{< param operator-extended-crd-version >}}
```

`param operator-extended-crd-version` is defined in config.toml so we don't have to hardcode `v1alpha2`